docs/source/tutorial.rst

```diff
@@ -42,14 +42,32 @@ To use a loader, (for example, ``urbansound8k``) you need to initialize it by ca
 You can indicate where the data would be stored and access by passing a path to ``data_home``, as explained below. Now ``us8k`` is a ``Dataset``
 object containing common methods, described in the following.
 
+Some datasets have more that one version available. Soundata can handle more than one version if included in the dataloader.
+
+.. code-block:: python
+
+    import soundata
+    dataset = soundata.initialize('urbansound8k', data_home='choose_where_data_live', version="1.0")
+
+By default, version is set to ``'default'``, which routes to the latest version available in the dataloader. 
+You may run ``soundata.list_dataset_versions('urbansound8k')`` to see all available versions of the dataset that are available through its dataloder.
```

Comment:
```suggestion
You may run ``soundata.list_dataset_versions('urbansound8k')`` to see all available versions of the dataset, that are available through its dataloder.
```

Reply:
Ok! I am waiting until we resolve the versioning issue here: https://github.com/soundata/soundata/pull/169#discussion_r1616019129, since it'll most likely make this line change again :)